cli/src/commands/global_config.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+use global_error::prelude::*;
+
+use crate::util::{global_config, paths, struct_fmt};
+
+#[derive(Parser)]
+pub enum SubCommand {
+	/// Reads the project's config
+	#[clap()]
+	ReadProject {
+		#[clap(long, value_parser)]
+		format: struct_fmt::Format,
+	},
+	/// Prints the path to the global config
+	#[clap()]
+	Path {},
```

Comment:
Does this need to be a enum struct variant, or could it just be named?


```suggestion
	Path,
```